pkgs/development/python-modules/cartopy/default.nix

```diff
@@ -0,0 +1,46 @@
+{ buildPythonPackage, lib, fetchPypi
+, pytest, filelock, mock, pep8
+, cython, isPy37, glibcLocales
+, six, pyshp, shapely, geos, proj, numpy
+, gdal, pillow, matplotlib, pyepsg, pykdtree, scipy, owslib, fiona
+}:
+
+buildPythonPackage rec {
+  pname = "cartopy";
```

Comment:
```suggestion
  pname = "Cartopy";
```

Reply:
Ehm, this would basically undo another change that was requested earlier by @worldofpeace .
It starts to feel like this review is not converging :D

Reply:
You're referring to https://github.com/NixOS/nixpkgs/pull/50951#discussion_r236031890. I must admit I probably never read that section of the manual. Personally, I don't think package names are of any importance compared to attribute names and I'd still prefer a single `pname`. What does @FRidh think?

Reply:
Judging by https://github.com/costrouc/nixpkgs-dev-tools/pull/3/commits/1953ac72be9cd4fdc8ee31be16c7999812c91a15, it seems like we should indeed normalize `pname`s.